linter.sh

```diff
@@ -1,14 +1,23 @@
 #!/usr/bin/env bash
 
-gometalinter ./... 2>&1 | read; [ $? == 1 ]
+if [ -d $GOPATH/src/github.com/alecthomas/gometalinter/  ]; then
+    echo "gometalinter already exists, not downloading."
```

Comment:
It is better not to print this because for each run, it will print this. When it is ok, it is enough to print `code format is ok!`.